utils/nctl/sh/scenarios/gov96.sh

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env bash
+
+source "$NCTL"/sh/scenarios/common/itst.sh
+
+# Exit if any of the commands fail.
+set -e
+
+#######################################
+# Runs an integration tests that tries to simulate
```

Comment:
```suggestion
# Runs an integration test that tries to simulate
```

Reply:
Fixed in [`070855e` (#1853)](https://github.com/casper-network/casper-node/pull/1853/commits/070855e155d41077944491d7afe5debdc3aace4f)